### PR TITLE
Added asyncio producer example

### PR DESCRIPTION
**Pull request by mhowlett**
To merge after the blog post is out (with url updated).

### CODE REVIEW

**Comment by edenhill** [comment]
Don't forget to flake8

**Comment by edenhill** [comment]
See CI for flake8 errors:
https://travis-ci.org/confluentinc/confluent-kafka-python/jobs/615145124#L3566

File: examples/asyncio.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Companion code to the blog post "Integrating Kafka With Python
+# Asyncio Web Applications"
+# https://www.confluent.io/blog/[path-to-blog-post]
+
+# Example Siege [https://github.com/JoeDog/siege] test:
+# siege -c 400 -r 200 'http://localhost:8000/items1 POST {"name":"testuser"}'
```
Comment (edenhill) [inline examples/asyncio.py]: prefer `$ siege ..` to make it evident it is a command.

File: examples/asyncio.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+import confluent_kafka
+from confluent_kafka import KafkaException
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from time import time
+from threading import Thread
+import uvicorn
+
+
+class AIOProducer:
```
Comment (edenhill) [inline examples/asyncio.py]: doc strings wouldnt hurt

File: examples/asyncio.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+        def ack(err, msg):
+            if err:
+                self._loop.call_soon_threadsafe(result.set_exception, KafkaException(err))
+            else:
+                self._loop.call_soon_threadsafe(result.set_result, msg)
+        self._producer.produce(topic, value, on_delivery=ack)
+        return result
+
+    def produce2(self, topic, value, on_delivery):
```
Comment (edenhill) [inline examples/asyncio.py]: two produce methods with the same name (numbers dont count) definitely need doc strings

File: examples/asyncio.py [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        raise HTTPException(status_code=500, detail=ex.args[0].str())
+
+
+@app.post("/items5")
+async def create_item5(item: Item):
+    return {"timestamp": time()}
+
+
+if __name__ == '__main__':
+    uvicorn.run(app, host='0.0.0.0', port=8000)
```
Comment (edenhill) [inline examples/asyncio.py]: 127.0.0.1 is safer

File: examples/asyncio.py (hunk 2 above)
Comment (mhowlett) [reply examples/asyncio.py]: yep. this omission caused by writing fatigue...

**Comment by mhowlett** [comment]
flake8 errors are due to type annotations. I'm disabling flake8 for the entire file rather than scattering lots of noqa's everywhere since I don't want to draw attention to this - it's beside the point and some people won't know what it is for.

**Pull request merged**